### ISSUE TITLE
Ticking Entity Crash, Animania As Possible Cause

### ISSUE THREAD

**Issue by Hypocrita20XX** [capnkirok/animania#66]
Good morning!

First off, I would like to thank you for such a nice mod! When I can get everything working, it's just awesome.
Second, I'm getting a consistent ticking entity crash at seemingly random times while exploring a world. I'll include the full crash log in this post. If any other information is needed, just let me know!

[crash-2017-03-20_06.38.31-server.txt](https://github.com/capnkirok/animania/files/854555/crash-2017-03-20_06.38.31-server.txt)

PS: I'm not 100% sure that this is an Animania issue per-say, but it is what comes up in the logs. This may just be a bad combination of mods. At this point, I'm not really sure what's going on, but figured it best to report this.

**Comment by capnkirok** [comment]
Thanks for the crash report. Looks like an issue with the Bull when it is in attack mode. Could be a compat issue, but I can look for a workaround. Thanks!

**Comment by capnkirok** [comment]
Ok, I've put in a null pointer check here, hopefully that solves the problem. The only way this could have occurred is if a player hit the bull, but the bull returned a null value when it tried to find the player.

**Comment by Hypocrita20XX** [comment]
That is so strange, when I was walking around, I didn't interact with them at all. Literally I was walking around taking in the sights, seeing how the terrain generated. Could FPS lag cause oddness like this? I've had a long-standing issue with that in 1.10.2 and have never have figured out the cause.

**Comment by capnkirok** [comment]
Maybe... not sure. I've also seen some weirdness with Entities in 1.10.2... not picking up the correct NBT status and then crashing (only on servers, tho)

**Comment by Hypocrita20XX** [comment]
Very strange. Oh, completely forgot about this until just now, but I did edit the sound files so that they're mono, reason being so I could hear the sounds properly in conjunction with the Sound Physics mod. Could this have any impact on this issue?

**Comment by Hypocrita20XX** [comment]
I think I finally found out what's going on, to a certain degree at least. It looks to be a mod incompatibility with the Harder Farming mod. I ripped out most of the mods and added the core terrain generation ones alongside the ones that I know affect mobs. After flying around for about 10 - 15 minutes with just Animania installed as a mob alteration mod, I experienced no crashes. As soon as I added in Harder Farming, it crashed within just a few minutes, showing virtually the same issue as the one I first posted about. I'll be adding in the other mods momentarily sans Harder Farming, see if anything else pops up, but hopefully this helps out a bit!
